packages/wrangler/src/__tests__/ci.test.ts

```diff
@@ -0,0 +1,84 @@
+import { mockConsoleMethods } from "./helpers/mock-console";
+import { useMockIsTTY } from "./helpers/mock-istty";
+import { runInTempDir } from "./helpers/run-in-tmp";
+import { runWrangler } from "./helpers/run-wrangler";
+import writeWranglerToml from "./helpers/write-wrangler-toml";
+
+const ENV_COPY = process.env;
+mockConsoleMethods();
+runInTempDir();
+
+afterEach(() => {
+  process.env = ENV_COPY;
+});
```

Comment:
If that is your preference, otherwise the file becomes the suite, including isolated scoping i.e. `beforeEach` `afterEach` to the file, and the suite name is inferred from the file.